src/modules/rangeproof/tests_impl.h

```diff
@@ -162,6 +162,11 @@ void test_rangeproof(void) {
     int j;
     int k;
     secp256k1_rand256(blind);
+    CHECK(secp256k1_pedersen_commit(ctx, commit, blind, 5));
```

Comment:
Can you add a comment above these lines explaining that you're checking that a when the commit value and claimed commit value don't match, the resulting signature is invalid?